### PR TITLE
Update CHANGELOG for the async loaded function

### PR DESCRIPTION
**Pull request by qistoph**
As requested in #556, I have update CHANGELOG for the async loaded function.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [2.1.1] - Unreleased
+- Add loaded function to modules, providing an async callback.
```
Comment (roramirez) [inline CHANGELOG.md]: This function should will be into the next feature release  2.2.0 not the 2.1.1 We the follow the Semantic Versioning. Is a new feature not a bug fix.

Comment (MichMich) [inline CHANGELOG.md]: @roramirez It will be 2.1.1, 2.2.0 will be introduced when there are breaking changes (just like 2.1.0).

**Comment by MichMich** [comment]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (roramirez) [inline CHANGELOG.md]: @MichMich its ok but this its not follow the [Semantic Versioning](http://semver.org/). This a new feature.

What happend if we need release a new version with only bugfixes? The version name will 2.1.0.1? Will the first 2 always be maintained?

**Pull request merged**